apps/genetics/public/matomo/init_matomo.js

```diff
@@ -0,0 +1,37 @@
+var _paq = window._paq || [];
+/* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+
+(function() {
+  function registerUserVisit() {
+    getLoggedInUser()
+      .then(function(user) {
+        _paq.push(['setUserId', user]);
+      })
+      .finally(function() {
+        _paq.push(['setCustomUrl', window.location.href]);
+        _paq.push(['setDocumentTitle', window.document.title]);
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+      });
+  }
+  var pushState = history.pushState;
+  history.pushState = function() {
+    pushState.apply(history, arguments);
+    registerUserVisit();
+  };
+  registerUserVisit();
+})();
+
+(function() {
+  var u = 'https://localhost:9443/';
```

Comment:
Can you make this url configurable in the `.env`? Ofc we are not always deploying on localhost

Reply:
It should be feasible by either inserting the URL into `apps/genetics/public/config.js` (like other settings) and reading from the global `window` object here, or, if this runs _before_ `config.js`, leaving a placeholder word to insert it right here, as this code file is in `public/` and served to the browser un-transpiled.

Reply:
Should be done now.